### ISSUE TITLE
How to configure Exq to push jobs to Sidekiq?

### ISSUE THREAD

**Issue by mhdatie** [akira/exq#488]
Hey, 

I have a requirement to push jobs from an Elixir server (Exq would be a client) to be processed by a Rails server configured with a Sidekiq server. They both share the same Redis instance. So far I have seen examples of how this would work the other way around where Exq is preforming the job enqueued by a Rails app

I have the `namespace` declared on the elixir side == `""` so that `Exq` doesn't take it into consideration, and I have no namespace declared on the Rails side

The `Exq.Enqueuer.enqueue(Exq.Enqueuer, "my_queue", "MyWorker", [])` is always expecting a worker to be declared

What I have done is that I have declared an empty list for `queues` and declared a Sidekiq worker for `my_queue`. The reason that I declared no queues on the `Exq` side is that I don't want it to be picked up by `Exq`

Unfortunately, I am unable to see the jobs being picked up. I am wondering if there is someone who has run into a similar challenge.

**Comment by ananthakumaran** [comment]
It's possible to create a job from Exq and run it on Rails, I have done it for many projects. I am guessing the `mode` [config](https://github.com/akira/exq#configuration) is not being set to `:enqueuer`, by default Exq runs the worker as well, which likely picks up the job before Sidekiq. You don't have to set up the queue or have a worker on Exq side.

**Comment by mhdatie** [comment]
I eventually figured it out and it was basically what you illustrated. I had to set the namespace to be `""` or `nil` to match the SIdekiq side. I didn't know that there is a way to only enable the `: enqueuer` (good to know) but what I did is set no queues at all `[]` and that made Exq not look for a worker. I then had the name of the worker match the Ruby Sidekiq class name and it worked. 

Closing this, thank you for the quick reply.
